<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iFoxtrot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelAddr">
        <property name="text">
         <string>IP/address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditAddr">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>192.168.1.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxPort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>5010</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAddr_2">
        <property name="text">
         <string>PLC address:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditPLCAddr">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>192.168.1.176</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="butConnect">
        <property name="text">
         <string>&amp;Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAutocon">
        <property name="text">
         <string>Autoconnect</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPLC">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListView" name="listViewItems"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelFoxName">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>5</number>
          </property>
          <widget class="QWidget" name="page_EMPTY"/>
          <widget class="QWidget" name="page_LIGHT">
           <widget class="QLabel" name="labelLight">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Light</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonLight">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>140</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Přepnout</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelLightStatus">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>70</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_RELAY">
           <widget class="QPushButton" name="pushButtonRelay">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>140</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Přepnout</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelRelay">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Relay</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelRelayStatus">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>70</y>
              <width>71</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_SHUTTER">
           <widget class="QLabel" name="labelShutter">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Shutter</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_DISPLAY">
           <widget class="QLabel" name="labelDisplay">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Display</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelDisplayUnit">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>130</y>
              <width>57</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxDisplayVal">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>130</y>
              <width>65</width>
              <height>25</height>
             </rect>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_SCENE">
           <widget class="QLabel" name="labelScene">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc1">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>50</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 1</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc2">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>80</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 2</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>110</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 3</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc4">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>140</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 4</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc5">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>170</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 5</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc6">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>200</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 6</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc7">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>230</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 7</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonSc8">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>260</y>
              <width>80</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Scene 8</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>676</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc7</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSc8</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonSceneClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonSceneClicked()</slot>
 </slots>
</ui>
